client/src/app/app.component.ts

```diff
@@ -13,18 +13,18 @@ const { version } = require('../../package.json');
   `,
   styles: [
     `
-    :host {
-      display: block;
-    }
-    ml-progress-bar {
-      height: 3px;
-      position: absolute;
-      top: 0;
-      left: 0;
-      right: 0;
-      z-index: 6;
-    }
-  `
+      :host {
+        display: block;
+      }
+      ml-progress-bar {
+        height: 3px;
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        z-index: 6;
+      }
+    `
```

Comment:
I don't think this change is coming from prettier, is it? Such a formatting change should go in a separate commit (I know this PR is still WIP, but just that we don't forget). Although I do believe this change is not meant to be.